sre-recipes/recipes/configs_based/README.md

```diff
@@ -0,0 +1,143 @@
+# Configs-Based SRE Recipe
+
+This directory contains configs for supported config-based SRE Recipes.
+
+## Writing the Config
+
+Each SRE Recipe config is a YAML file with the following overall structure.
+
+```yaml
+name: <Put the name of your recipe here. This is not user visible.>
+description: <Put the description of your recipe here. This is not user visible.>
+config:
+  # This section defines the actions to run when `sandboxctl sre-receipes break`
+  # is called" . In practice, this is where you try to break sandbox services.
+  #
+  # If your SRE Recipe has complex logic, prefer running a shell command to
+  # run a shell script instead.
+  #
+  # You can use, or contribute, more action templates by referring to the
+  # "Supported SRE Recipe Actions" below.
+  break:
+    # The `run-shell-commands` action template simply runs the given shell
+    # commands one at a time as it is.
+    - action: run-shell-commands
+      commands:
+        - echo 'first command'
+        - echo 'another command'
+
+    # The `loadgen-spawn` action template will spawn the given load shape
+    # producible by `user_type` (see `sre/loadgenerator/locust_tasks`) at a
+    # `spawn_rate` of users per second until it reaches a total of `user_count`
+    # users. The users will keep generating load until at least `stop_after`
+    # seconds have passed.
+    - action: loadgen-spawn
+      user_type: BasicHomePageViewingUser
```

Comment:
we should mention where user types are defined

Reply:
Done.